objects/src/notes/recipient.rs

```diff
@@ -54,6 +57,15 @@ impl NoteRecipient {
     pub fn digest(&self) -> Digest {
         self.digest
     }
+
+    /// Returns the recipient encoded as [Felt]s.
+    pub fn to_felts(&self) -> Vec<Felt> {
```

Comment:
minor nit: we usually name such methods `to_elements()`.

Reply:
changed